redshells/model/graph_convolutional_matrix_completion.py

```diff
@@ -408,21 +528,53 @@ def _make_graph(self) -> GraphConvolutionalMatrixCompletionGraph:
             ignore_item_embedding=self.ignore_item_embedding)
 
     @staticmethod
-    def _eliminate(matrix: sp.csr_matrix, user_indices, item_indices):
-        matrix = matrix.copy()
-        # `lil_matrix` is too slow
-        matrix[list(user_indices), list(item_indices)] = 0
-        matrix.eliminate_zeros()
-        return matrix
+    def load(file_path: str) -> str:
```

Comment:
It seems this function doesn't return `str`.

Reply:
I've fixed the type hinting.